code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -157,3 +157,12 @@
 		JUSTICE_ATTRIBUTE = 40
 	)
 
+/obj/item/clothing/suit/armor/ego_gear/courage
+	name = "courage"
+	desc = "Their weapons did not hurt me. but when I looked back and you were not there, it felt as if I was on the verge of death."
```

Comment:
"Their weapons did not hurt me. but when I looked back and you were not there, it felt as if I was on the verge of death." to "Their weapons did not hurt me, but when I looked back and you were not there; It felt as if I was on the verge of death."